### ISSUE TITLE
git clone of repo fails, pip install lama-stack does not provide llama cli tool

### ISSUE THREAD

**Issue by bsenftner** [meta-llama/llama-stack#127]
Attempting to git clone this repo fails with no public read permission.

Doing a "pip install llama-stack" appears to work, but there is no llama cli tool afterwards. 

So, this repo appears to be a nop, can't get anything to work at all.

**Comment by bsenftner** [comment]
I manually downloaded the repo and tried the manual install steps given in the README. This error is emitted:

ERROR: file:///home/bsenftner/meta/llama-stack-main/llama_stack does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

**Comment by bsenftner** [comment]
The README instructions are wrong, they say to run:

 $CONDA_PREFIX/bin/pip install -e .

From the llama_stack directory, which is wrong, it should be run one directory above.

**Comment by yanxi0830** [comment]
Yes, the directory to run `pip install -e .` is in the `llama-stack` directory, not `llama-stack/llama_stack`.

**Comment by kvchitrapu** [comment]
I installed in a venv and llama cli installed fine.

**Comment by nathan-weinberg** [comment]
@bsenftner is this still an issue with the latest doc updates?
